src/d3d/nvapi_d3d_low_latency_device.cpp

```diff
@@ -118,7 +135,27 @@ namespace dxvk {
         return d3dLowLatencyDevice;
     }
 
-    LowLatencyFrameIdGenerator* NvapiD3dLowLatencyDevice::GetFrameIdGenerator(IUnknown* device) {
+    Com<ID3DLowLatencyDevice> NvapiD3dLowLatencyDevice::GetLowLatencyDevice(ID3D12CommandQueue* commandQueue) {
```

Comment:
Some duplication with the other `GetLowLatencyDevice` and also same map, but I don't see a more elegant way either.

Reply:
At worst we could extract common parts to another method and pass the way to extract `ID3DLowLatencyDevice` in another parameter as something like `std::function` but I feel like that would be greatly overengineered solution. Duplication, as long as it's limited like this, seems more readable.

Reply:
Yes, let’s keep your version, it won’t get better :)